### DISCUSSION TITLE
Welcome to faker-file Discussions!

### DISCUSSION THREAD

**Discussion by barseghyanartur** [barseghyanartur/faker-file#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by martinburchell** [comment]
Hi I'm Martin. I develop CRATE (a set of tools to anonymise and extract text from clinical records) within the University of Cambridge Department of Psychiatry. See https://github.com/ucam-department-of-psychiatry/crate

We're using faker-file in our tests. At some point I intend to spin out some common providers from CRATE (and our other project CamCOPS) into a faker-medical extension.

**Reply by barseghyanartur** [reply]
That's wonderful, Martin! I'm glad to see you here.
